### ISSUE TITLE
Add new functions

### ISSUE THREAD

**Issue by MikyGonzalez** [keebus/koji#2]
Add values() keys() pairs() split() reduce() functions? Also add string utility functions like get substring, probably can access strings like array and read his value, but not can modify it?

```
var strtoarray = def (str) {
    ...
}

var str = "Hello world.";
var array_of_str = strtoarray(str); # Or even native: str.to_array()?
```

**Comment by keebus** [comment]
Yes, obviously those functions are all planned. Not sure if they'll be free function as in "len(str)" or method calls. I'll have to think about it.
